output/cloud/expv2/output.go

```diff
@@ -218,6 +284,21 @@ func (o *Output) flushMetrics() {
 	o.logger.WithField("t", time.Since(start)).Debug("Successfully flushed buffered samples to the cloud")
 }
 
+// flushRequestMetadatas periodically flushes traces collected in RequestMetadatasCollector using flusher.
+func (o *Output) flushRequestMetadatas() {
+	start := time.Now()
+
+	ctx, cancel := context.WithTimeout(context.Background(), o.config.TracesPushInterval.TimeDuration())
+	defer cancel()
+
+	err := o.requestMetadatasFlusher.flush(ctx)
+	if err != nil {
+		o.logger.WithError(err).Error("Failed to push trace samples to the cloud")
```

Comment:
```suggestion
		o.logger.WithError(err).WithField("t", time.Since(start)).Error("Failed to push trace samples to the cloud")
```
In case of error it could even more relevant